src/external/IntelRDFPMathLib20U2/LIBRARY/src/bid128_to_int64.c

```diff
@@ -2022,8 +2022,9 @@ if ((C1.w[1] > 0x0001ed09bead87c0ull)
       // shift right C* by Ex-128 = bid_shiftright128[ind]
       shift = bid_shiftright128[ind - 1];	// 0 <= shift <= 102
       if (ind - 1 <= 21) {	// 0 <= ind - 1 <= 21
-	Cstar.w[0] =
-	  (Cstar.w[0] >> shift) | (Cstar.w[1] << (64 - shift));
+	if (shift > 0) {
+	  Cstar.w[0] = (Cstar.w[0] >> shift) | (Cstar.w[1] << (64 - shift));
+	}

```

Comment:
`shift == 0` is a no-op, but one which was UB because it did a left shift by 64.